### DISCUSSION TITLE
Side panel (Edge panel) application for Android

### DISCUSSION THREAD

**Discussion by technodrk** [FossifyOrg/General-Discussion#166] [Ideas]
Hello, an edge panel application like samsung one ui on some phones can be good because this feature provides ease of use and functionality, but many phones do not have this feature and there is literally no safe and properly working application that provides this feature. 
https://www.youtube.com/watch?v=PktVzyPqliA
![edge-panels](https://github.com/FossifyOrg/General-Discussion/assets/153454840/0d2263bc-f9de-4f63-9360-bb5814ad5f2d)
![the-edge-panels](https://github.com/FossifyOrg/General-Discussion/assets/153454840/d9a46235-a3ae-40c4-b3da-f02459ca7777)

**Comment by naveensingh** [comment]
Apart from the PDF viewer (yet to be released, it was separated from the File Manager so it's technically not a new project), no new Fossify apps are planned. 

**Reason:** There is enough to do as it is, properly maintaining even one or two medium to complex apps requires at least a part-time commitment, we have 18 apps in total.

**Reply by technodrk** [reply]
Yes, it may be difficult to integrate all fossify applications, but in my opinion, the main feature of the edge panel is that you can see other applications when you scroll the panel, I think that this feature is enough, if nothing else, such a program can be developed.

**Reply by technodrk** [reply]
I hope that one day you will start developing new applications, this application could be good, and anyway thank you for your answer and for all the applications you have developed.
